src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/Target_ARM/ARMEmitter.cs

```diff
@@ -32,12 +61,44 @@ public void EmitPUSH(Register reg)
             Builder.EmitShort((short)(0x0d04 + ((byte)reg << 12)));
         }
 
+        // pop reg
+        public void EmitPOP(Register reg)
+        {
+            Builder.EmitByte(0x5d);
+            Builder.EmitByte(0xf8);
+            Builder.EmitShort((short)(0x0b04 + ((byte)reg << 12)));
+        }
+
         // mov reg, reg
+        // reg range: 0-7
         public void EmitMOV(Register destination, Register source)
         {
             Builder.EmitShort((short)(0x4600 + (((byte)destination & 0x8) << 4) + (((byte)source & 0x8) << 3) + (((byte)source & 0x7) << 3) + ((byte)destination & 0x7)));
         }
 
+        // ldr reg, [reg]
+        // reg range: 0-7
```

Comment:
In the outdated comment for mov, I wrote about ldr. It's thumb ldr with offset, really 3 bits for registers. You can check. 0x68 is not just opcode, it's opcode with zero offset. I wanted extend this emitting with offset and generalize it with another ldr.w later.